lib/fluent/plugin/output.rb

```diff
@@ -534,6 +552,35 @@ def handle_stream(tag, es)
         meta_and_data.keys
       end
 
+      def handle_stream_with_standard_format(tag, es)
+        meta_and_data = {}
+        records = 0
```

Comment:
I see.

For optimization phase, we should separate the code into using `es.events` and each count mode.
Or we should calculate # of events in the initialization phase of MessagePackEventStream.
This is the out scope of this patch.